tsconfig.base.json

```diff
@@ -16,6 +16,7 @@
     "noUncheckedIndexedAccess": true,
     "noUnusedLocals": false,
     "noUnusedParameters": false,
+    "removeComments": true,
```

Comment:
does it keep license comments?

Reply:
oh, not really, so another solution is required.